utils/apiRequests.ts

```diff
@@ -8,6 +8,8 @@ const giftsBaseUrl = '/api/gifts';
  * @returns an array that contains all the gifts as objects
  */
 export async function getAllGifts() {
+  // tässä esimerkiksi käytetty sleeppiä, jotta voi kokeilla miten käyttäytyy 3 sekunnin fetchin takia
+  //await sleep(3000);
```

Comment:
Jätin tämän vain esimerkiksi, että missä esim. `sleep`-funktiota on käytetty

Tähän saattaa olla jokin kirjasto, jolla voi testata hidasta nettiä, mutta tämä on nyt ainakin toiminut ihan hyvin

Reply:
Yksi tapa testata hidasta nettiä on Chromen developer toolsin Network-tabilta löytyvä "Throttling"-asetus.
![image](https://github.com/user-attachments/assets/45d1b7bc-b55b-401f-ba14-5db7898b18ac)

Reply:
Tuo on hyvä! :+1: 

Näin jälkikäteen oon käyttänyt `sleep`iä backendissä simuloimaan hidasta / mahdollisia hitaita tilanteita backendissa. Esim. että lahjojen haussa kestäisikin vaikka 4 sekuntia. Sen avulla oon testannut, miten latausindikaattori ja `tanstack` yleisesti on toiminut

Tuo Chromen oma netinrajotin on parempi clienttipuolen testaamiseen :+1: 

Jos tuon `utilFunctions.ts`-tiedoston säilyttää niin ainakin siirto jonnekin eripaikkaan lienee hyvä, koska ei ole frontissa tarvetta